### PR TITLE
Add dockerfile for easy usage.

### PR DESCRIPTION
**Pull request by dakulov**
I made these changes for myself. But I think it can be useful for someone.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM node:10
```
Comment (pgiraud) [inline Dockerfile]: I'd suggest using `node:10-slim`.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+COPY ./ /app
+WORKDIR /app
+RUN npm install && npm run build
+
+FROM nginx
```
Comment (pgiraud) [inline Dockerfile]: I'd suggest using `nginx:alpine`.

**Comment by pgiraud** [comment]
Thanks @dakulov 
See my suggestions.

**Comment by dakulov** [comment]
@pgiraud thank you. I've fixed.

**Pull request merged**